O/oneAPI_Level_Zero/oneAPI_Level_Zero_Loader/build_tarballs.jl

```diff
@@ -52,3 +52,4 @@ dependencies = [
 
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
                preferred_gcc_version=v"8")
```

Comment:
This is for a future version, but it'd be good to require julia v1.6, so that the dependency on Pkg will be automatically dropped.